core/worker/src/test/java/com/cognifide/aet/worker/drivers/JSErrorsOnChromeWebDriverTest.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Automated Exploratory Tests
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.cognifide.aet.worker.drivers;
+
+import com.cognifide.aet.job.api.collector.JsErrorLog;
+import com.cognifide.aet.worker.drivers.chrome.LogEntryToJsError;
+import com.google.common.collect.Ordering;
+import java.util.Set;
+import java.util.TreeSet;
+import java.util.logging.Level;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.logging.LogEntries;
+import org.openqa.selenium.logging.LogEntry;
+import org.openqa.selenium.logging.LogType;
+import org.openqa.selenium.logging.LoggingPreferences;
+import org.openqa.selenium.remote.CapabilityType;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+@Ignore
```

Comment:
Please add here message why it it ignored: `@Ignore("Why this test is ignored")`

Reply:
True I didn't noticed that while C/P from:
https://github.com/Cognifide/aet/blob/feature/chrome-as-main-aet-browser/core/worker/src/test/java/com/cognifide/aet/worker/drivers/JSErrorsOnChromeWebDriverTest.java

I have to catch-up with Witek about this. Maybe I'll remove this class from codebase

Reply:
`feature/chrome-as-main-aet-browser` branch was create only for PoC purpose.
I think this test was never meant to be merged to `master`.

Reply:
ok, I will remove this file from PR